tests/components/airnow/test_config_flow.py

```diff
@@ -0,0 +1,168 @@
+"""Test the AirNow config flow."""
+from pyairnow.errors import AirNowError, InvalidKeyError
+
+from homeassistant import config_entries, data_entry_flow, setup
+from homeassistant.components.airnow.const import DOMAIN
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_LATITUDE,
+    CONF_LONGITUDE,
+    CONF_NAME,
+    CONF_RADIUS,
+)
+
+from tests.async_mock import patch
+
+CONFIG = {
+    CONF_API_KEY: "abc123",
+    CONF_LATITUDE: 34.053718,
+    CONF_LONGITUDE: -118.244842,
+    CONF_RADIUS: 75,
+    CONF_NAME: "Home",
+}
+
+# Mock AirNow Response
+MOCK_RESPONSE = [
+    {
+        "DateObserved": "2020-12-20",
+        "HourObserved": 15,
+        "LocalTimeZone": "PST",
+        "ReportingArea": "Central LA CO",
+        "StateCode": "CA",
+        "Latitude": 34.0663,
+        "Longitude": -118.2266,
+        "ParameterName": "O3",
+        "AQI": 44,
+        "Category": {
+            "Number": 1,
+            "Name": "Good",
+        },
+    },
+    {
+        "DateObserved": "2020-12-20",
+        "HourObserved": 15,
+        "LocalTimeZone": "PST",
+        "ReportingArea": "Central LA CO",
+        "StateCode": "CA",
+        "Latitude": 34.0663,
+        "Longitude": -118.2266,
+        "ParameterName": "PM2.5",
+        "AQI": 37,
+        "Category": {
+            "Number": 1,
+            "Name": "Good",
+        },
+    },
+    {
+        "DateObserved": "2020-12-20",
+        "HourObserved": 15,
+        "LocalTimeZone": "PST",
+        "ReportingArea": "Central LA CO",
+        "StateCode": "CA",
+        "Latitude": 34.0663,
+        "Longitude": -118.2266,
+        "ParameterName": "PM10",
+        "AQI": 11,
+        "Category": {
+            "Number": 1,
+            "Name": "Good",
+        },
+    },
+]
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+    assert result["errors"] == {}
+
+    with patch(
+        "pyairnow.WebServiceAPI._get",
+        return_value=MOCK_RESPONSE,
+    ) as mock_lat_long:
```

Comment:
Please also patch the integration setup functions on all create entry results. See our scaffold script template test code for example.